Apps/Src/SendCarCAN.c

```diff
@@ -48,19 +51,27 @@ uint8_t get_SendCarCAN_Q_Space(void) {
 
 /**
  * @brief Wrapper to put new message in the CAN queue
- * @param message the CAN message to put in SendCarCAN Queue
 */
 void SendCarCAN_Put(CANDATA_t message){
-    OS_ERR err;
+    OS_ERR err = OS_ERR_NONE;
     CPU_TS ticks;
-
-    OSMutexPend(&CarCAN_Mtx, 0, OS_OPT_PEND_BLOCKING, &ticks, &err);
-    assertOSError(err);
+    bool success = false;
+    
+    static uint8_t carcan_ctr = 0;
     
-    bool success = SendCarCAN_Q_put(&CANFifo, message);
+    if(carcan_ctr > 3){
```

Comment:
This isn't too important, but maybe a macro could be used instead of 3

Reply:
Yes, will create a macro for this.